Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/ActionHttpHandler.java

```diff
@@ -45,19 +42,14 @@ public void handle(HttpExchange httpExchange) throws IOException {
         if (path.contains("yarnui")) {
             JobUtils.openYarnUIHistory(clusterConnectString, requestDetail.getAppId());
         } else if (path.contains("sparkui")) {
-
-                try {
-                    Application application = JobViewCacheManager.getSingleSparkApplication(new ApplicationKey(requestDetail.getCluster(), requestDetail.getAppId()));
-                    JobUtils.openSparkUIHistory(clusterConnectString, requestDetail.getAppId(), application.getLastAttemptId());
-
-                } catch (ExecutionException e) {
-                    e.printStackTrace();
-                }
-
-
+            try {
+                Application application = JobViewCacheManager.getSingleSparkApplication(new ApplicationKey(requestDetail.getCluster(), requestDetail.getAppId()));
+                JobUtils.openSparkUIHistory(clusterConnectString, requestDetail.getAppId(), application.getLastAttemptId());
+                JobUtils.setResponse(httpExchange, "open browser successfully");
+            } catch (ExecutionException e) {
+                JobUtils.setResponse(httpExchange, "open browser error", 500);
```

Comment:
Is it possible to add more error message in response? Such as the details of the exception. For now, if user send back the error message, we could not figure out the root cause of browser open failed.